tempest/src/test/kotlin/app/cash/tempest/LogicalDbTransactionTest.kt

```diff
@@ -195,11 +195,9 @@ class LogicalDbTransactionTest {
     // Introduce a race condition.
     musicTable.playlistInfo.save(playlistInfoV2)
 
-    assertThatIllegalStateException()
-      .isThrownBy {
-        musicDb.transactionWrite(writeTransaction)
-      }
-      .withCauseExactlyInstanceOf(TransactionCanceledException::class.java)
+    assertFailsWith<TransactionCanceledException> {
```

Comment:
could assert that the message/reasons are what we expect, but that's mostly a nice to have

Reply:
I did try to do that but it was very brittle so I leaned towards leaving it out for now